File: alpaul_gls/Content/Downloadable/PHAO2018092520181018161350/Business/Pais.vb
Imports System.Collections.Generic

Public Class Pais
    Public Shared Function Add(value As Models.Pais) As Integer
        Return Data.Pais.Add(value)
    End Function

    Public Shared Function Edit(value As Models.Pais) As Integer
        Return Data.Pais.Edit(value)
    End Function

    Public Shared Function Delete(value As String) As Integer
        Return Data.Pais.Delete(value)
    End Function

    Public Shared Function GetById(value As String) As Models.Pais
        Return Data.Pais.GetById(value)
    End Function

    Public Shared Function GetByGUID(value As String) As Models.Pais
        Return Data.Pais.GetByGUID(value)
    End Function

    Public Shared Function GetAll() As List(Of Models.Pais)
        Return Data.Pais.GetAll()
    End Function
End Class

File: alpaul_gls/Content/Downloadable/PHAO2018092520181018171302/Business/Pais.cs
using System.Collections.Generic;

namespace Business
{
    public class Pais
    {
	    private static Data.Pais obj = new Data.Pais();

        public static int Add(Models.Pais value)
        {
            return obj.Add(value);
        }

        public static int Edit(Models.Pais value)
        {
            return obj.Edit(value);
        }

        public static int Delete(stirng value)
        {
            return obj.Delete(value);
        }

        public static Models.Pais GetById(string ID)
        {
            return obj.GetById(ID);
        }

		public static Models.Pais GetByGUID(string ID)
        {
            return obj.GetByGUID(ID);
        }

        public static List<Models.Pais> GetAll()
        {
            return obj.GetAll();
        }
    }
}using System.Collections.Generic;

namespace Business
{
    public class Pais
    {
	    private static Data.Pais obj = new Data.Pais();

        public static int Add(Models.Pais value)
        {
            return obj.Add(value);
        }

        public static int Edit(Models.Pais value)
        {
            return obj.Edit(value);
        }

        public static int Delete(stirng value)
        {
            return obj.Delete(value);
        }

        public static Models.Pais GetById(string ID)
        {
            return obj.GetById(ID);
        }

		public static Models.Pais GetByGUID(string ID)
        {
            return obj.GetByGUID(ID);
        }

        public static List<Models.Pais> GetAll()
        {
            return obj.GetAll();
        }
    }
}